core/src/empty/empty.tsx

```diff
@@ -0,0 +1,30 @@
+import s from "./empty.module.css";
+import { createElement } from "react";
+import { Button } from "../button/button";
+import cloudImg from "./empty1.jpg";
+
+interface EmptyProps {
+    active: boolean,
+    errorMsg: string,
```

Comment:
Just "message" would be better. Empty doesn't always mean "error". It could be perfectly fine